packages/vuetify/src/components/VDataTable/VSimpleTable.sass

```diff
@@ -95,7 +100,11 @@
 
   // Account for scroll bar
   .v-table__header, .v-data-footer
```

Comment:
@nekosaur `v-table__header` exists only here - should it be removed or it's a typo in a class name?

Reply:
It is most likely a remnant of an earlier iteration of the refactor (specifically another virtual rows implementation I think). So should be safe to remove.